services/app/src/components/views/quest/cardtemplates/decision/ResolveDecision.tsx

```diff
@@ -36,7 +36,10 @@ export default function resolveDecision(props: Props): JSX.Element {
     } as Record<keyof typeof Outcome, string>)[outcome];
     return (
       <Card title={title} theme={props.theme} inQuest={true}>
-        <Callout icon="adventurer_white">{instruction}</Callout>
+        {(instruction)
+          ? <Callout icon="adventurer_white">{instruction}</Callout>
```

Comment:
Is this meant to be a white adventurer icon all the time? Shouldn't it depend on if it's in combat or not?

Reply:
As discussed, added a TODO for later implementation.